docs/docs/handling-events.md

```diff
@@ -2,7 +2,115 @@
 id: handling-events
 title: Handling Events
 permalink: docs/handling-events.html
-prev: state-and-lifecycle.html
 ---
 
-TODO
+Handling events with React elements is very similar to handling events on DOM elements. There are some syntactic differences:
+
+* React events are named using camelCase, rather than lowercase
+* With JSX you pass a function as the event handler, rather than a string
+
+For example, the HTML:
+
+```html
+<button onclick="activateLasers()">Activate Lasers</button>
```

Comment:
The problem with ignoring HTML is that we don't really expect people to read the React documentation to figure out what events all the elements support. We only super-briefly discuss it. I think really, to know what events there are, you have to be aware of the rule, "it's just like HTML except camelcased".